### PR TITLE
Feature/header links

### PR DESCRIPTION
**Pull request by ThibaultNocchi**
This PR adds links to the header items. Please test a header with episodes. It's supposed to have a link to the episode and a link to the show.

### CODE REVIEW

**Comment by neopc10** [comment]
This is super helpful. The logos for shows, artists, and movies aren't clickable. It looks like the intention is to include those.

**Comment by ThibaultNocchi** [comment]
It should be good, I only kept the episode name going to the episode page for now :)

**Review by MrTimscampi — APPROVED** [review]

File: client/components/Layout/HomeHeader/HomeHeaderAlbumTitle.vue [hunk 1]
```diff
@@ -1,35 +1,45 @@
 <template>
   <div>
-    <v-img
+    <nuxt-link
       v-if="imageTag"
```
Comment (ferferga) [inline client/components/Layout/HomeHeader/HomeHeaderAlbumTitle.vue:4]: Might be good to set a check for ``item.AlbumArtists[0]`` here as well?

File: client/components/Layout/HomeHeader/HomeHeaderEpisodeTitle.vue [hunk 2]
```diff
@@ -1,40 +1,50 @@
 <template>
   <div>
-    <v-img
+    <nuxt-link
       v-if="imageTag"
```
Comment (ferferga) [inline client/components/Layout/HomeHeader/HomeHeaderEpisodeTitle.vue:4]: Same

**Review by ferferga — APPROVED** [review]

**Pull request merged**